types/react-phone-number-input/index.d.ts

```diff
@@ -184,6 +184,12 @@ export interface PhoneInputProps extends Omit<React.InputHTMLAttributes<string>,
      */
     international?: boolean;
 
+    /**
+     * If an initial value is passed, and initialValueFormat property is not set, then the initial value
+     * is formatted in international format.
+     */
+    initialValueFormat?: string;
```

Comment:
Per the [prop types](https://gitlab.com/catamphetamine/react-phone-number-input/-/blob/master/source/PhoneInputWithCountry.js#L584) it looks like the only allowable value is 'national' rather than any arbitrary string.

```
initialValueFormat?: 'national';
```

Reply:
Tnx @jlismore I've added a deprecation comment. And updated initialValueFormat to "national" only.